File: csci24000_spring2017_A3/HorseJava/Player.java
//I pledge that I have neither given nor recieved help on this assigment
import java.util.*;
import java.io.*;


//Creates player class
public class Player
{
	private int score = 0;
	private char[] string;
	//constructor
	public Player()
		{
			this.string = new char[6];
		}
	//generates random number for shot
	public int shot()
		{
			Random rand = new Random();
			int a = rand.nextInt(2) + 1;
			return a;
		}
	//Getters for score
	public int getscore()
		{
			return this.score;
		}
	public char[] getstring()
		{
			return this.string;
		}
	//Setters for string and score
	public void setstring()
		{
			char[] str = {'H','O','R','S','E'};
			int x = getscore();
			this.string[x] = str[x];
		}
	public void setscore()
		{
			this.score = score + 1;
		}
	// eliminates garbage characters by initializing the array to spaces
	public void initstring()
		{
			int t;
			for(t=0;t<6;t++)
				{
					this.string[t] = ' ';
				}
		}
}

File: csci24000_spring2017_A3/HorseC++/Player.cpp
//Honor Pledge: I pledge that I have neither given nor received any help on this assignment
#include "Player.h"
#include <stdlib.h>
#include <ctime>
char str[6] = "HORSE";
//constructor
player::player()
{
	player::string = new char [6];
}
//generates chance to make the shot
int player::shot()
{
	int a;
	a = rand() %2;
	return a;
}
//Getters
int player::getscore()
{
	return player::score;
}
char *player::getstring()
{	
	return player::string;
}
//setters
void player::setscore()
{
	this->score = score + 1;
}
void player::setstring()
{
	int x = getscore();
	this->string[x] = str[x];
}
//initializes the string to contain spaces
void player::initstring()
{
	int t;
	for(t=0;t<6;t++)
		{
			this->string[t] = ' ';
		}
}

//Destructor
player::~player()
{
	delete[] this->string;
}